### PR TITLE
Add get school experience page

### PR DESCRIPTION
**Pull request by EmmaFrith**
The new Steps iteration won't include a section on school experience so we need a space for Get school experience content.

Need some formatting help - have temporarily used the 'get an adviser' call to action side bar component but think that icon is reserved for getting an adviser content. 

Have also used an existing banner that's used elsewhere - we may want to use something different.

### CODE REVIEW

File: app/views/content/get-school-experience/get-school-experience.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Be flexible with your availability if possible and persevere with your search.
+
+## Maths or physics teaching internships
+
+[Find a paid teaching internship if you’re doing a STEM undergraduate degree](https://getintoteaching.education.gov.uk/teaching-internship-providers).
+
+## If you cannot get into the classroom right now
+
+[Observe teachers’ lessons and analysis of their lessons on the Oak National Academy website](https://teachers.thenational.academy/lessons-for-itt).
```
Comment (peteryates) [inline app/views/content/get-school-experience/get-school-experience.md:57]: This page looks a bit broken. Can we link to a YouTube channel or something instead?

Comment (EmmaFrith) [reply app/views/content/get-school-experience/get-school-experience.md:57]: Couldn't agree more - I think it's the specific videos on this page though that we've been asked to link to... :/

File: app/views/content/get-school-experience/get-school-experience.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+[Find schools near you](https://get-information-schools.service.gov.uk/) and give them a ring to find out who to talk to about getting school experience.
+
+It's normal to have to try a few schools before you find one that can help.
+
+They're not obliged to offer experience and may be limiting visitors due to coronavirus (COVID-19).
+
+They also need to be careful about allowing visitors into the school for the safety of children and young people.
+
+You may be asked to have a DBS check to ensure you're safe to work with children.
```
Comment (peteryates) [inline app/views/content/get-school-experience/get-school-experience.md:47]: Are these two statements close enough to be combined?

Comment (EmmaFrith) [reply app/views/content/get-school-experience/get-school-experience.md:47]: combined as in put into one paragraph or turned into one sentence?

Comment (peteryates) [reply app/views/content/get-school-experience/get-school-experience.md:47]: Either really, they just felt like they overlapped a bit.

File: app/views/content/get-school-experience/get-school-experience.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+title: Get school experience
+image: "/assets/images/international-dt.jpg"
+mobileimage: "/assets/images/steps-hero-mob.jpg"
+description: |-
+  School experience can help your teacher training application. Here's how to get school experience.
+date: "2021-05-04"
+right_column:
+  ctas:
+    - title: Find school experience
+      link_text: "Request early years, primary or secondary experience."
```
Comment (MylesJarvis) [inline app/views/content/get-school-experience/get-school-experience.md:11]: remove full stop.

File: app/views/content/get-school-experience/get-school-experience.md (hunk 2 above)
Comment (EmmaFrith) [reply app/views/content/get-school-experience/get-school-experience.md:47]: true dat - will roll into one

File: app/views/content/get-school-experience/get-school-experience.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* [request early years, primary or secondary experience on GOV.UK](https://schoolexperience.education.gov.uk/) - not all schools advertise experience here and some only offer online experience due to coronavirus (COVID-19) restrictions
+* approach schools yourself
+
+## Approach schools yourself
+
+[Find schools near you](https://get-information-schools.service.gov.uk/) and give them a ring to find out who to talk to about getting school experience.
+
+It's normal to have to try a few schools before you find one that can help.
+
+They're not obliged to offer experience and may be limiting visitors due to coronavirus (COVID-19).
```
Comment (MylesJarvis) [inline app/views/content/get-school-experience/get-school-experience.md:43]: Similar to Pete's query below - are these two lines not combine-able? They feel v similar.

File: app/views/content/get-school-experience/get-school-experience.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+To find experience you can:
+
+* [request early years, primary or secondary experience on GOV.UK](https://schoolexperience.education.gov.uk/) - not all schools advertise experience here and some only offer online experience due to coronavirus (COVID-19) restrictions
+* approach schools yourself
+
+## Approach schools yourself
+
+[Find schools near you](https://get-information-schools.service.gov.uk/) and give them a ring to find out who to talk to about getting school experience.
+
+It's normal to have to try a few schools before you find one that can help.
```
Comment (MylesJarvis) [inline app/views/content/get-school-experience/get-school-experience.md:41]: Similar to Pete's query below - are these two lines not combine-able? They feel v similar.

File: app/views/content/get-school-experience/get-school-experience.md (hunk 1 above)
Comment (MylesJarvis) [reply app/views/content/get-school-experience/get-school-experience.md:57]: Yep - this is a policy steer on where we should link to

File: app/views/content/get-school-experience/get-school-experience.md (hunk 4 above)
Comment (EmmaFrith) [reply app/views/content/get-school-experience/get-school-experience.md]: have just pushed another commit - see what you think.. 

trying to keep sentences really short and paragraphs to the shortest possible length, whilst also being really clear about why things might not be available

**Review by peteryates — APPROVED** [review]
I think this is a good start. The final sentence (on events) feels a little vague but overall it's clear and informative.

**Pull request merged**